src/PythonConverter/PythonWorker.ts

```diff
@@ -37,14 +36,19 @@ checkIfConverterReady()
 class PythonWorkerBase implements PythonWorker {
 	readonly isPythonWorker: true = true;
 	async initPyodide(onReady: () => void) {
-		const pyodide = await self.loadPyodide();
+		const pyodide = await import(
+			'https://cdn.jsdelivr.net/pyodide/v0.25.0/full/pyodide.js'
```

Comment:
use this opportunity to match pyodide version with the one from package file (I think its 0.26 somthing...)